File: calc.c
#include <stdio.h>
int choise;

void timestables(){
    int table = 0;
    int amount = 0;
    printf("enter the times table you want\n");
    scanf("%d", &table);
    for (size_t i = 0; i < 12; i++)
    {   
        amount = table * i;
        printf("%d * %d = %d\n",table , i , amount);
    }
    
    
}

void vat(){
    float vatamount;
    printf("Enter the amount for vat to be added\n ");
    scanf("%f", &vatamount);
    vatamount = vatamount + vatamount*0.2;
    printf("Your product with vat added is %f\n", vatamount);
    
    
}

void tax(){
    float taxamount;
    float income;
    printf("Enter the amount your income is\n ");
    scanf("%f", &income);
    if (income <= 12570)
    {
        printf("you pay no tax");
    }else if (income >= 12571 && income < 50270)
    {
        taxamount = income * 0.2;
        printf("you need to pay %f amount of tax.\n", taxamount);
    }
    else if (income >= 50271 && income < 150000)
    {
        taxamount = income * 0.4;
        printf("you need to pay %f amount of tax.\n", taxamount);
    }
    else if (income >= 150000 )
    {
        taxamount = income * 0.45;
        printf("you need to pay %f amount of tax.\n", taxamount);
    }
    
    
    
   
    
    
}


int main(){
    while (1)
    {
        printf("press 1 for times tables\n");
        printf("press 2 for vat calc\n");
        printf("press 3 for taxs calc\n");
        scanf("%d", &choise);
        if (choise == 1)
        {
            timestables();

        }else if (choise == 2)
        {
            vat();
        }
        else if (choise == 3)
        {
            tax();
        }
        
    }
    
    
    
    
    return 0;
}

File: calc.py
def timestable():
    x = int(input(" Please enter the times table you want to know: "))
    for i in range(12):
        print(f"{x} * {i} = {x*i}")
    print()
    main()


def vat():
    x = float(input(" Please enter price so vat can be added: "))
    x = x + x * 0.2
    print(f" Your price with vat is now {x}")
    print()
    main()


def taxs():
    x = float(input("Enter your salary"))
    if x < 12570:
        print("your total tax amount if $0")
    elif x >= 12571 and x < 50270:
        print(f"your tax amount on {x} is {x * 0.2}")
    elif x >= 50271 and x < 150000:
        print(f"your tax amount on {x} is {x * 0.4}")
    elif x >= 150000:
        print(f"your tax amount on {x} is {x * 0.45}")
        print()
    main()


def main():
    print(
        """
    press 1 for times tables
    press 2 for vat calc
    press 3 for taxs calc
    """
    )
    option = int(input())
    if option == 1:
        timestable()
    elif option == 2:
        vat()
    elif option == 2:
        taxs()
    else:
        print("please enter a valid input")
        main()


main()
